### ISSUE TITLE
'ChartViewDelegate' is unavailable: cannot find Swift declaration for this class.

### ISSUE THREAD

**Issue by JW00332** [danielgindi/Charts#1613]
I've just updated to Swift 3 and implemented the master branch of Charts, the rest of my code is now working in Swift 3, but I am getting the error in the title, as well as errors such as "PieChartView" is unavailable. I am using the most recent version, does anyone have any idea what I have missed, I have checked the "migrating to swift3" and I can't find anything that would fix my problem?

Thanks in advance, for taking the time to help me out!

EDIT: I have dragged the charts xcodeproject as in all previous implementations and embedded the binary etc. It is strange as auto complete recognises things such as "PieChartView" etc.

**Comment by petester42** [comment]
I don't what to say since I just tried it in the demo and it works. Just embed charts like the demo and it will work.

**Comment by JW00332** [comment]
I've just tried it in a "fresh" Xcode project, and it works fine. Its only an issue when trying to migrate an older Xcode project to Xcode 8. The project builds on "generic devices" but not on simulator/device. Again, the "fresh" implementation works fine. What can I do to make my older project play nice with the new version? Are there cached files i should delete? I have already cleared derived data/ clean etc.

Thanks,

James
